src/pages/HomePage.tsx

```diff
@@ -1,12 +1,34 @@
 import { Link } from 'react-router-dom';
+
 import styled from 'styled-components';
 
+import { useLocalStorage } from 'usehooks-ts';
+
+import { apiService } from '../services/ApiService';
+
 function HomePage() {
+  const [, setUserType] = useLocalStorage('userType', '');
```

Comment:
p4: 사용하지 않는 코드라는 것을 명시적으로 해주기 위해서 아래와 같이 쓰기도 해요. `userType` 문자열도 상수화가 되면 좋을 것 같네요..!

```suggestion
  const [_, setUserType] = useLocalStorage('userType', '');
```

Reply:
https://github.com/kkkkkSE/chatchat-frontend/pull/22#discussion_r1245042645 이쪽에 리뷰들 달았는데 수정되면 좋을 것 같아요